test/browser_tests/shared_objects/careersSocialSection.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var _socialSection =
+element( by.css( '[data-qa-hook="social-section"]' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/25449812700f8ec8d3b478bc53f2cad34387e630